example/builder/src/basic.over_react.g.dart

```diff
@@ -158,10 +52,12 @@ abstract class _$$BasicProps extends _$BasicProps
 
   /// The default namespace for the prop getters/setters generated for this class.
   @override
-  String get propKeyNamespace => 'BasicProps.';
+  String get propKeyNamespace => '';
```

Comment:
FIXME @greglittlefield-wf this should remain for shorthand syntax. Also maybe we should deprecate this?